src/Commands/LightsOn.h

```diff
@@ -18,9 +19,16 @@ class LightsOn : public frc::Command {
 	bool IsFinished() override;
 	void End() override;
 	void Interrupted() override;
+	void SetTimeFlashing(double timeFlashing);
 private:
 	frc::Notifier blink;
 	bool blinking;
 	void Notify();
+	frc::Notifier flash;
+	bool flashed;
```

Comment:
`flash` and `flashed` are not needed. `blink` and `blinking` provide the same functionality.